### ISSUE TITLE
Readme points to missing file libmpsse.dll

### ISSUE THREAD

**Issue by olivierbloch** [WildernessLabs/Meadow.Samples#32]
In the ReadMe section for Windows.Desktop samples, instructions indicate to include the DLL for the FT232H expander in the project, but the link is dead and the said library is nowhere to be found in the repo

**Comment by jorgedevs** [comment]
Good catch, @olivierbloch. That is not longer needed and I have updated the readme for that section. Will be available on the next release.
